### PR TITLE
fixes missing border under copy permissions & buttons placement

### PR DESCRIPTION
**Pull request by genti94**
+ created new file to match the file structure of the django-cms sass components

**before** 
screen shot 2016-04-27 at 09 58 31

**after
screen shot 2016-04-27 at 09 58 48
**

### CODE REVIEW

File: djangocms_admin_style/sass/components/_dialog.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// resets form and background styles in dialog window #186
+// fixes missing border under copy permissions & text & paddings/margins of the buttons
```
Comment (lory87) [inline djangocms_admin_style/sass/components/_dialog.scss]: can you please add pr id here?

**Comment by vxsx** [comment]
can you add a changelog entry? "Fixed copy permissions dialog view" or smth similar

codewise LGTM

**Comment by lory87** [comment]
@genti94 Can you please add change log entry?

**Comment by lory87** [comment]
👍

**Pull request merged**